### PR TITLE
feat: agent token encryption

### PR DESCRIPTION
**Pull request by KulkarniShashank**
### What ###
This refers to a security measure where sensitive information, such as authentication tokens or API keys, is encrypted before being stored in a database.
### Why ###
Encrypting tokens adds an extra layer of security.
### How ###
Update the codebase to include a mechanism for token encryption before storing it in the database.
Utilize cryptographic libraries or functions to perform the encryption securely.

### CODE REVIEW

**Review by nishad-ayanworks — APPROVED** [review]
LGTM

File: apps/agent-service/src/agent-service.controller.ts [hunk 1]
```diff
@@ -99,34 +97,34 @@ export class AgentServiceController {
...
+    orgId: string;
   }): Promise<IAgentProofRequest> {
-    return this.agentServiceService.sendProofRequest(payload.proofRequestPayload, payload.url, payload.apiKey);
+    return this.agentServiceService.sendProofRequest(payload.proofRequestPayload, payload.url, payload.orgId);
   }
   //DONE
   @MessagePattern({ cmd: 'agent-verify-presentation' })
-  async verifyPresentation(payload: { url: string; apiKey: string }): Promise<IPresentation> {
-    return this.agentServiceService.verifyPresentation(payload.url, payload.apiKey);
+  async verifyPresentation(payload: { url: string; orgId: string }): Promise<object> {
```
[the remark this answers is not in the document]
Comment (KulkarniShashank) [reply apps/agent-service/src/agent-service.controller.ts:115]: I included the object type to create a common function for making calls to microservices, known as a nats call.

File: apps/agent-service/src/agent-service.controller.ts [hunk 2]
```diff
@@ -99,34 +97,34 @@ export class AgentServiceController {
...
-  async getConnections(payload: { url: string; apiKey: string }): Promise<object> {
-    return this.agentServiceService.getConnections(payload.url, payload.apiKey);
+  async getConnections(payload: { url: string; orgId: string }): Promise<object> {
+    return this.agentServiceService.getConnections(payload.url, payload.orgId);
   }
 
   @MessagePattern({ cmd: 'agent-get-connection-details-by-connectionId' })
-  async getConnectionsByconnectionId(payload: { url: string, apiKey: string }): Promise<IConnectionDetailsById> {
-    return this.agentServiceService.getConnectionsByconnectionId(payload.url, payload.apiKey);
+  async getConnectionsByconnectionId(payload: { url: string, orgId: string }): Promise<object> {
```
[the remark this answers is not in the document]
Comment (KulkarniShashank) [reply apps/agent-service/src/agent-service.controller.ts:126]: I included the object type to create a common function for making calls to microservices, known as a nats call.

File: apps/agent-service/src/agent-service.controller.ts [hunk 3]
```diff
@@ -149,97 +147,92 @@ export class AgentServiceController {
...
   }): Promise<object> {
-    return this.agentServiceService.sendOutOfBandProofRequest(payload.proofRequestPayload, payload.url, payload.apiKey);
+    return this.agentServiceService.sendOutOfBandProofRequest(payload.proofRequestPayload, payload.url, payload.orgId);
   }
 
   //DONE
   @MessagePattern({ cmd: 'get-agent-verified-proof-details' })
-  async getVerifiedProofDetails(payload: { url: string; apiKey: string }): Promise<IProofPresentationDetails[]> {
-    return this.agentServiceService.getVerifiedProofDetails(payload.url, payload.apiKey);
+  async getVerifiedProofDetails(payload: { url: string; orgId: string }): Promise<object> {
```
[the remark this answers is not in the document]
Comment (KulkarniShashank) [reply apps/agent-service/src/agent-service.controller.ts:157]: I included the object type to create a common function for making calls to microservices, known as a nats call.

**Review by tipusinghaw — APPROVED** [review]

**Pull request merged**